js/client-chat.js

```diff
@@ -1247,6 +1247,11 @@
 					if (!this.tournamentBox.parseMessage(row.slice(1), row[0] === 'tournaments')) break;
 					// fallthrough in case of unparsed message
```

Comment:
I tested it in this position and it worked fine, however, for readability, moving it up is probably a good idea.

Thanks!